### PR TITLE
rename ggplot2 and plyr episodes

### PR DESCRIPTION
**Pull request by raynamharris**
renames the ggplot2 and plyr episodes in the Rmd files and the guide as per suggestions in issue #366

### CODE REVIEW

File: _episodes_rmd/12-plyr.Rmd [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Split-Apply-Combine
+title: Splitting and Combining Data Frames with dplyr
```
Comment (naupaka) [inline _episodes_rmd/12-plyr.Rmd]: So it looks like I was wrong. This is a `plyr` lesson, not `dplyr`. Can you make that adjustment?

Comment (raynamharris) [reply _episodes_rmd/12-plyr.Rmd]: Good catch! These too names are too similar!

**Comment by naupaka** [comment]
Thanks!

**Comment by naupaka** [comment]
Closes #366

**Pull request merged**